src/tools/rust-analyzer/crates/base-db/src/fixture.rs

```diff
@@ -283,6 +283,8 @@ impl ChangeFixture {
             }
         }
 
+        let _ = file_id;
```

Comment:
I'm probably missing something, but why is this needed?

Reply:
`file_id.0` is updated in the `if !proc_macro_names.is_empty() {` branch above, but is never read again. This is flagged as an unused assignment, correctly.